src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -603,8 +608,17 @@ services:
 
     form.type.shop.traffic_seo.meta.url_schema:
         class: 'PrestaShopBundle\Form\Admin\Configure\ShopParameters\TrafficSeo\Meta\UrlSchemaType'
+        parent: 'form.type.translatable.aware'
+        public: true
         arguments:
```

Comment:
Couldn't figure out why this argument was here in the first place, I guess it was somesort of leftover.